arrow-data-source/common/pom.xml

```diff
@@ -5,7 +5,7 @@
     <parent>
         <artifactId>spark-arrow-datasource</artifactId>
         <groupId>com.intel.oap</groupId>
-        <version>1.4.0-SNAPSHOT</version>
+        <version>1.3.1</version>
         <relativePath>../pom.xml</relativePath>
```

Comment:
Initially I tried to modify the version in 1.3.1 branch only. 
https://github.com/oap-project/gazelle_plugin/commit/0cee28a3eb015fcd7cd7d518feff47efa5d70275